src/python/foglamp/admin_api/server.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import asyncio
+from foglamp.admin_api import controller as admin_api_controller
+ 
+ 
+def start():
+    """Starts the service"""
+    admin_api_controller.start()
+    asyncio.get_event_loop().run_forever()
+
```

Comment:
"admin api" and "device" run in different processes now. "admin api" will be deleted -- but the UI has a dependency on it.